app/models/miq_replication_worker/runner.rb

```diff
@@ -36,7 +36,7 @@ def valid_destination_settings?
     dest_conf = worker_settings.fetch_path(:replication, :destination)
     dest_conf[:adapter] ||= "postgresql"
     region = MiqRegion.my_region
-    host_user_pass = dest_conf.values_at(:host, :port, :username, :password, :database, :adapter)
+    host_user_pass = dest_conf.values_at(:host, :username, :password, :database, :adapter)
     host_user_pass.none?(&:blank?) && MiqRegionRemote.region_valid?(region.guid, region.region, *host_user_pass)
```

Comment:
If someone has changed the port, then the value won't get passed along to `.region_valid?` here.